extensions/ql-vscode/src/query-history.ts

```diff
@@ -635,7 +636,7 @@ export class QueryHistoryManager extends DisposableObject {
     const variantAnalysisRemovedSubscription = this.variantAnalysisManager.onVariantAnalysisRemoved(async (variantAnalysis) => {
       const items = this.treeDataProvider.allHistory.filter(i => i.t === 'variant-analysis' && i.variantAnalysis.id === variantAnalysis.id);
```

Comment:
Just wondering about it: can there be multiple items? With the decision to remove the historyId there should always only be one history item per variant analysis?